app/models/service.rb

```diff
@@ -0,0 +1,14 @@
+class Service < ApplicationRecord
+    belongs_to :area
+
+    #validations
+    validates_associated :area
+    validates :name, presence: true
+    validates :name, format: { with: /\A[a-zA-Z]+\z/}   
+    validates :classification, presence: true
+    validates :classification, format: { with: /\A[a-zA-Z]+\z/}   
+    validates :purchase_price, presence: true
+    validates :purchase_price, numericality: true
+    validates :selling_price, presence: true
+    validates :selling_price, numericality: true
```

Comment:
When doing multiple validations of the same field with `validates`. I think (if I'm not mistaken) that you can one line them all